### ISSUE TITLE
Add a UI display panel to show a diff between two objects' features

### ISSUE THREAD

**Issue by lichtefeld** [isi-vista/adam#1069]
For M4's demonstration we'll need to be able to show which features of an object are unique to that object and which are shared. Assuming we want this functionality to be available for any two objects in any arbitrary scene the following needs to be implemented:
1. A way to select up-to two objects to display in this "comparative" mode
2. A three-panel layout where one panel lists features unique to object 1, the middle panel lists the shared features, and a third panel which lists the unique features of object 3. 
3. Ideally changing one object's selection from the list (say there are 3 objects in the scene) should dynamically update this comparison display.

This feature implementation is important for M4 but is currently blocked by ADAM providing the relevant features to the UI in a reasonable way to allow for this calculation of uniqueness.

**Comment by lichtefeld** [comment]
We will add this field to the API return within the post_decode output.

```json
{
  "differences_panel": {
     "table": ["feature_1", "feature_2"],
     "Similarities": ["Category(white)"],
     "cup": ["feature_3", "features_4"],
  }
}
```

**Comment by lichtefeld** [comment]
Closed by #1080
